app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    _, file_name, new_name = command.split()
```

Comment:
Consider adding a check to ensure that the command contains exactly three elements and starts with 'cp'. This will prevent errors due to malformed commands.